backend/danswer/connectors/confluence/connector.py

```diff
@@ -399,7 +298,6 @@ def __init__(
 
         # Remove trailing slash from wiki_base if present
         self.wiki_base = wiki_base.rstrip("/")
```

Comment:
we remove self.space because the only place its used outside of this function is in the metadata. See the comment therer for more details